src/LauncherV3/MainViewModel.cs

```diff
@@ -145,7 +149,8 @@ private bool CanUpdate()
     }
 
     public ICommand StartUpdateCrpgCommand => IsGameUpToDate ? StartCrpgCommand : UpdateGameFilesCommand;
-    public string StartUpdateText => IsGameUpToDate ? "Launch cRPG" : "Check For Update";
+    public string StartUpdateText => IsGameUpToDate ? "Launch cRPG" : "Update cRPG";
+    public LinearGradientBrush StartUpdateBackground => IsGameUpToDate ? new LinearGradientBrush(Color.FromArgb(50, 148, 56, 55), Color.FromArgb(255, 148, 56, 55), 90) : new LinearGradientBrush(Color.FromArgb(50, 251, 214, 1), Color.FromArgb(255, 251, 214, 1), 90);
 
```

Comment:
MainViewModel should not have UI logic in MVVM pattern. 
This should be moved to XAML and binded to mainviewmodel
ChatGPT proposal 
```
<UserControl.Resources>
    <LinearGradientBrush x:Key="StartCrpgButtonBackground" StartPoint="0,0" EndPoint="1,1">
        <GradientStop Color="#328A28" Offset="0"/>
        <GradientStop Color="#205A18" Offset="1"/>
    </LinearGradientBrush>
    <LinearGradientBrush x:Key="UpdateCrpgButtonBackground" StartPoint="0,0" EndPoint="1,1">
        <GradientStop Color="#FBD602" Offset="0"/>
        <GradientStop Color="#B89500" Offset="1"/>
    </LinearGradientBrush>
</UserControl.Resources>

```

```
<Button Content="{Binding StartUpdateText}"
        Background="{StaticResource ResourceKey={Binding StartUpdateBackgroundResourceKey}}" />
```

Reply:
I've moved the logic into the xaml.